### PR TITLE
feat/info

### PR DESCRIPTION
**Pull request by fdlk**

### CODE REVIEW

File: test.R [hunk 1]
```diff
@@ -1,7 +1,8 @@
-library(DSI)
 library(DSMolgenis)
 library(dsBaseClient)
 
+dsGetInfo(molgenis())
```
Comment (tommydeboer) [inline test.R:4]: How should this work? I get an error: `Error in molgenis() : could not find function "molgenis"`

Comment (tommydeboer) [reply test.R:4]: This does work:
`dsGetInfo(new("MolgenisDriver"))`

File: R/MolgenisDriver.R [hunk 2]
```diff
@@ -110,8 +115,13 @@ setMethod(
...
 #' @export
-setMethod("dsGetInfo", "MolgenisDriver", function(dsObj, ...) { # nolint
-  # TODO implement
-})
+methods::setMethod(
+  "dsGetInfo", "MolgenisDriver",
+  function(dsObj, ...) { # nolint
+    return(list(
+      driver.version = packageVersion("DSI"),
+      client.version = packageVersion("DSMolgenis")
```
Comment (tommydeboer) [inline R/MolgenisDriver.R:124]: Shouldn't these two be the other way around?

Comment (fdlk) [reply R/MolgenisDriver.R:124]: owh.. unsure! looking it up

**Pull request merged**